src/pages/home/report/ReportActionItem.js

```diff
@@ -117,6 +120,21 @@ function ReportActionItem(props) {
         focusTextInputAfterAnimation(textInputRef.current, 100);
     }, [isDraftEmpty]);
 
+    // Hide the message if it is being moderated for a higher offense, or is hidden by a moderator
+    // Removed messages should not be shown anyway and should not need this flow
+    useEffect(() => {
+        if (_.isEmpty(props.action.moderationDecisions)) {
```

Comment:
Let's first check that it's an ADDCOMMENT report action and then update to check the message object like we talked about